docs/V1/Customer/get_respondents.md

```diff
@@ -1,176 +1,395 @@
-* *approximateKeyword (`String`)* : Chaîne de caractère recherchée dans le nom et l'adresse mail des candidats.
-* * *from (`Int`)* : UNIX timestamp. Permet de récupérer les candidats à partir d'une date donnée
+### Réponse
+`200` - La liste des candidats de l'entreprise au format JSON
+```json
+{
+  "items": [
+    {
+      "questionnaire": {
+        "id": {questionnaireId},
+        "name": "{questionnaireName}",
+        "slug": "{questionnaireSlug}",
+        "offer_ref": {questionnaireReference}
+      },
+      "score_total_percentage": null,
+      "id": {respondentId},
+      "score_auto": null,
+      "score_total": null,
+      "main_comment": null,
+      "status": {respondentStatus},
+      "phone_number": "{respondentPhoneNumber}",
+      "email": "{respondentEmail}",
+      "first_name": "{respondentFirstName}",
+      "last_name": "{respondentLastName}",
+      "candidate_link": "http://fr.scoringline.com/admin/{slugEnt}/questionnaire/{questionnaireSlug}/repondant/{respondentId}/show",
+      "custom_status": {
+        "id": {customStatusId},
+        "name": "{customStatusName}",
+        "slug": "{customStatusSlug}"
+      },
+      "synthesis_link": "http://fr.scoringline.com/admin/{slugEnt}/questionnaire/{questionnaireSlug}/repondant/{respondentId}/generate/synthesis.pdf"
+    },
+    {
+      "questionnaire": {
+        "id": {questionnaireId},
+        "name": "{questionnaireName}",
+        "slug": "{questionnaireSlug}",
+        "offer_ref": {questionnaireReference}
+      },
+      "score_total_percentage": null,
+      "id": {respondentId},
+      "score_auto": null,
+      "score_total": null,
+      "main_comment": null,
+      "status": {respondentStatus},
+      "phone_number": "{respondentPhoneNumber}",
+      "email": "{respondentEmail}",
+      "first_name": "{respondentFirstName}",
+      "last_name": "{respondentLastName}",
+      "candidate_link": "http://fr.scoringline.com/admin/{slugEnt}/questionnaire/{questionnaireSlug}/repondant/{respondentId}/show",
+      "custom_status": {
+        "id": {customStatusId},
+        "name": "{customStatusName}",
+        "slug": "{customStatusSlug}"
+      },
+      "synthesis_link": "http://fr.scoringline.com/admin/{slugEnt}/questionnaire/{questionnaireSlug}/repondant/{respondentId}/generate/synthesis.pdf"
+    }
+  ],
+  "total_results": 11,
+  "current_page": "2",
+  "nextPage": "http://api.scoringline.com/api-saas/v1/customer/respondents.json?&page=3",
+  "previousPage": "http://api.scoringline.com/api-saas/v1/customer/respondents.json&page=1"
+}
+```
 
-# Cas pratique
+### Description des données
+Nom           | Type         | Description
+--------------|--------------|---------------------------------------------------------------------------------
+items         | object       | Objet représentant un répondant (voir [Objet`items`](#objet-items))
```

Comment:
Add a single space between "Objet" and "`items`". You have the same problem everywhere actually.